### PR TITLE
dollarshaveclub/fastboot:4.0.0

### PR DESCRIPTION
**Pull request by briangonzalez**
Features:

- Pinned Yarn version
- Remove dependency on forked fastboot-app-server until PR lands (https://github.com/ember-fastboot/fastboot-app-server/pull/35)
- Move static asset options to a custom middleware

### CODE REVIEW

File: scripts/ci/test.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+docker build -t dollarshaveclub/fastboot . && \
```
Comment (arjansingh) [inline scripts/ci/test.sh:3]: `!#/bin/bash`

**Review by arjansingh — COMMENTED** [review]
Configure express static to GZIP responses.

`  1) gzips responses
  5 passing (247ms)
  1 failing
  1) dollarshaveclub/fastboot gzips responses:
     AssertionError: expected undefined to equal 'gzip'
      at request.then.response (docker-container-test.js:52:57)
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:510:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:567:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:612:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:691:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
`

File: scripts/ci/test.sh (hunk 1 above)
Comment (briangonzalez) [reply scripts/ci/test.sh:3]: Nope – `#!/bin/sh`

Comment (briangonzalez) [reply scripts/ci/test.sh:3]: @arjansingh – I am going to merge this since I got your go-ahead in person.

Comment (arjansingh) [reply scripts/ci/test.sh:3]: yeah this is good otherwise.

**Pull request merged**